docs/UserGuide.md

```diff
@@ -722,7 +731,7 @@ return to [Table of Contents](#table-of-contents)
 
 ### Viewing help : `help`
 
-Receives instruction on how to access EduTrack's user guide if you need assistance while using the app.
+Displays a pop-up window containing the link to the EduTrack user guide. You can use this command to quickly access the user guide.
```

Comment:
Sounds good! Maybe it sounds better if we refer as "this user guide" instead of "the EduTrack user guide" since the user is viewing from this user guide

Reply:
Good suggestion! Updated!